Makefile

```diff
@@ -148,14 +106,7 @@ test:
 	find . -name ".coverage*" -not -name ".coveragerc" -exec rm -fr "{}" \;
 
 .PHONY: all-checks
-all-checks:
-	make clean \
-	&& make formatters \
-	&& make code-checks \
-	&& make security \
-	&& make generators \
-	&& make common-checks-1 \
-	&& make common-checks-2
+all-checks: clean formatters code-checks security make generators make common-checks-1 make common-checks-2
```

Comment:
Remove the extra `make` statements

Reply:
fixed in [d8e7885](https://github.com/valory-xyz/open-autonomy/pull/1214/commits/d8e7885f22025637be39086a427ded915e4b76ad)